src/services/analytics/gtm.ts

```diff
@@ -48,6 +48,23 @@ export const gtmSetChainId = (chainId: string): void => {
   _chainId = chainId
 }
 
+export const enum AbTest {
+  SAFE_CREATION = 'safe-creation',
+}
+
+let _abTest: AbTest | null = null
```

Comment:
Using a hashmap would require us to manually map every property manually. As such, I've opted for a localised setter like we use with `chainId`.

The example use case is explained in the testing example: set this in a `useEffect` on the given route, resetting it on cleanup.